package.json

```diff
@@ -39,14 +39,17 @@
         "@aws-cdk/aws-iam": "1.119.0",
         "@aws-cdk/aws-logs": "1.119.0",
         "@aws-cdk/aws-secretsmanager": "1.119.0",
-        "@aws-cdk/core": "1.119.0",
+        "@aws-cdk/core": "^1.119.0",
+        "@aws-cdk/lambda-layer-kubectl": "^1.119.0",
         "@aws-cdk/pipelines": "1.119.0",
         "@types/bcrypt": "^5.0.0",
         "aws-sdk": "^2.952.0",
         "bcrypt": "^5.0.1",
         "js-yaml": "3.13.1",
+        "object-utils": "^0.0.1-SNAPSHOT",
```

Comment:
Please remove these dependencies, we don't need them.

Reply:
addressed

Reply:
addressed